### PR TITLE
Update docs re pyenv-virtualenv setup

### PR DESCRIPTION
**Pull request by stevejalim**
Quick fix to documentation - the instructions did work, but only while the shell session persisted. This fix makes it persistent/re-triggered each new session

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -161,9 +161,9 @@ using ``zsh``, the ``pyenv`` docs have other routes ::
 
     $ brew install pyenv-virtualenv
 
-2. Configure your shell to init ``pyenv-virtualenv`` on start - again, this is noted in the ``pyenv-virtualenv`` project's `own docs <https://github.com/pyenv/pyenv-virtualenv>`_, in more detail ::
+2. Configure your shell to init ``pyenv-virtualenv`` on start - again, this is noted in the ``pyenv-virtualenv`` project's `own documentation <https://github.com/pyenv/pyenv-virtualenv>`_, in more detail. The following will slot in a command that will work as long as you have pyenv-virtualenv installed::
 
-    $ eval "$(pyenv virtualenv-init -)"
+    $ echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.zshrc
```
Comment (pmac) [inline docs/install.rst:166]: Do we assume somewhere that all users are using ZSH?

Comment (maureenlholland) [reply docs/install.rst:166]: yep, there's some text earlier in the page about the assumptions and how to find instructions for other setups:

> The following assumes you are on MacOS, using zsh as your shell and [Homebrew](https://brew.sh/) as your package manager. If you are not, there are installation instructions for a variety of platforms and shells in the READMEs for the two pyenv projects.

**Comment by stevejalim** [comment]
@pmac @maureenlholland Hold on merging this - i want to add a couple more quck things in an hour or two's time

**Comment by stevejalim** [comment]
@pmac @maureenlholland PR extended and unblocked

**Review by pmac — APPROVED** [review]
🥇

**Review by maureenlholland — APPROVED** [review]
LGTM r+ ✍️

**Pull request merged**